common/src/main/java/com/wynntils/features/user/CombatXPGainMessageFeature.java

```diff
@@ -0,0 +1,59 @@
+/*
+ * Copyright © Wynntils 2022.
+ * This file is released under AGPLv3. See LICENSE for full license details.
+ */
+package com.wynntils.features.user;
+
+import com.wynntils.core.config.Config;
+import com.wynntils.core.features.UserFeature;
+import com.wynntils.core.features.properties.FeatureInfo;
+import com.wynntils.core.notifications.NotificationManager;
+import com.wynntils.mc.event.ClientTickEvent;
+import com.wynntils.wynn.model.CharacterManager;
+import com.wynntils.wynn.utils.WynnUtils;
+import net.minecraftforge.eventbus.api.SubscribeEvent;
+
+@FeatureInfo
+public class CombatXPGainMessageFeature extends UserFeature {
+
+    @Config
+    public boolean getCombatXPGainMessages = true;
```

Comment:
In Artemis, we don't need configs like this. If you disable the feature, then you wont get combat xp gain messages :)

Reply:
ah i misunderstood that the last time you told me about it